test/fixtures/icu/localizationData-v74.2.json

```diff
@@ -0,0 +1,128 @@
+    "mr": "Fri Jul 25 1980 01:35:33 GMT+0100 (मध्‍य युरोपियन प्रमाण वेळ)",
+    "te": "Fri Jul 25 1980 01:35:33 GMT+0100 (సెంట్రల్ యూరోపియన్ ప్రామాణిక సమయం)"
+  },
+  "dateTimeFormats": {
+    "en": "7/25/1980, 1:35:33 AM",
+    "zh": "1980/7/25 01:35:33",
+    "hi": "25/7/1980, 1:35:33 am",
+    "es": "25/7/1980, 1:35:33",
+    "fr": "25/07/1980 01:35:33",
+    "ar": "٢٥‏/٧‏/١٩٨٠، ١:٣٥:٣٣ ص",
+    "bn": "২৫/৭/১৯৮০, ১:৩৫:৩৩ AM",
+    "ru": "25.07.1980, 01:35:33",
+    "pt": "25/07/1980, 01:35:33",
+    "ur": "25/7/1980، 1:35:33 AM",
+    "id": "25/7/1980, 01.35.33",
+    "de": "25.7.1980, 01:35:33",
+    "ja": "1980/7/25 1:35:33",
+    "pcm": "25/7/1980 01:35:33",
+    "mr": "२५/७/१९८०, १:३५:३३ AM",
+    "te": "25/7/1980 1:35:33 AM"
+  },
+  "dateFormats": {
+    "en": "7/25/1980",
+    "zh": "1980/7/25",
+    "hi": "25/7/1980",
+    "es": "25/7/1980",
+    "fr": "25/07/1980",
+    "ar": "٢٥‏/٧‏/١٩٨٠",
+    "bn": "২৫/৭/১৯৮০",
+    "ru": "25.07.1980",
+    "pt": "25/07/1980",
+    "ur": "25/7/1980",
+    "id": "25/7/1980",
+    "de": "25.7.1980",
+    "ja": "1980/7/25",
+    "pcm": "25/7/1980",
+    "mr": "२५/७/१९८०",
+    "te": "25/7/1980"
+  },
+  "displayNames": {
+    "en": "Switzerland",
+    "zh": "瑞士",
+    "hi": "स्विट्ज़रलैंड",
+    "es": "Suiza",
+    "fr": "Suisse",
+    "ar": "سويسرا",
+    "bn": "সুইজারল্যান্ড",
+    "ru": "Швейцария",
+    "pt": "Suíça",
+    "ur": "سوئٹزر لینڈ",
+    "id": "Swiss",
+    "de": "Schweiz",
+    "ja": "スイス",
+    "pcm": "Swítsaland",
+    "mr": "स्वित्झर्लंड",
+    "te": "స్విట్జర్లాండ్"
+  },
+  "numberFormats": {
+    "en": "275,760.913",
+    "zh": "275,760.913",
+    "hi": "2,75,760.913",
+    "es": "275.760,913",
+    "fr": "275 760,913",
+    "ar": "٢٧٥٬٧٦٠٫٩١٣",
+    "bn": "২,৭৫,৭৬০.৯১৩",
+    "ru": "275 760,913",
+    "pt": "275.760,913",
+    "ur": "275,760.913",
+    "id": "275.760,913",
+    "de": "275.760,913",
+    "ja": "275,760.913",
+    "pcm": "275,760.913",
+    "mr": "२,७५,७६०.९१३",
+    "te": "2,75,760.913"
+  },
+  "pluralRules": {
+    "en": "other",
+    "zh": "other",
+    "hi": "one",
+    "es": "other",
+    "fr": "one",
+    "ar": "zero",
+    "bn": "one",
+    "ru": "many",
+    "pt": "one",
+    "ur": "other",
+    "id": "other",
+    "de": "other",
+    "ja": "other",
+    "pcm": "one",
+    "mr": "other",
+    "te": "other"
+  },
+  "relativeTime": {
+    "en": "586,920.617 hours ago",
+    "zh": "586,920.617小时前",
+    "hi": "5,86,920.617 घंटे पहले",
+    "es": "hace 586.920,617 horas",
+    "fr": "il y a 586 920,617 heures",
+    "ar": "قبل ٥٨٦٬٩٢٠٫٦١٧ ساعة",
+    "bn": "৫,৮৬,৯২০.৬১৭ ঘন্টা আগে",
+    "ru": "586 920,617 часа назад",
+    "pt": "há 586.920,617 horas",
+    "ur": "586,920.617 گھنٹے پہلے",
+    "id": "586.920,617 jam yang lalu",
+    "de": "vor 586.920,617 Stunden",
+    "ja": "586,920.617 時間前",
+    "pcm": "586,920.617 áwa wé dọ́n pas",
+    "mr": "५,८६,९२०.६१७ तासांपूर्वी",
+    "te": "5,86,920.617 గంటల క్రితం"
+  }
+}
```

Comment:
Can we add an EOL at the EOF?

Reply:
Sure. Should this become a lint rule? Right now some json fixtures have EOL at the end, some do not.

Reply:
We do not have any lint rules in `test/fixtures`, and there are some fixtures files that are purposefully lacking an EOL, so until then we're relying on human reviews. I can recommend setting a setting in your editor to automatically add the EOL.

Reply:
Well, it wasn't related to the editor since the file was generated with `writeFile(JSON.stringify())`. Anyway, EOLs are added.